cli/lib/kontena/cli/nodes/update_command.rb

```diff
@@ -3,18 +3,42 @@ class UpdateCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "NODE_ID", "Node id"
+    requires_current_master
+    requires_current_master_token
+    requires_current_grid
+
+    parameter "NAME", "Node name"
+
+    option ["--token"], "TOKEN", "Node token"
+    option "--generate-token", :flag, "Generate new node token"
+
     option ["-l", "--label"], "LABEL", "Node label", multivalued: true
+    option "--clear-labels", :flag, "Clear node labels"
```

Comment:
The `kontena grid update` uses `--no-default-affinity` for something similar, should I change this to `kontena node update --no-labels`? The `--clear` just sounds more logical in this case.